pyproject.toml

```
@@ -28,24 +28,103 @@
   dependencies = []
 
   [project.urls]
-    Changelog     = "https://github.com/data-apis/array-api-typing/releases"
-    Repository    = "https://github.com/data-apis/array-api-typing"
+    Repository = "https://github.com/data-apis/array-api-typing"
+    Changelog  = "https://github.com/data-apis/array-api-typing/releases"
 
 
 [build-system]
-  requires      = ["hatch-vcs", "hatchling"]
+  requires = ["hatch-vcs", "hatchling"]
   build-backend = "hatchling.build"
 
 
 [dependency-groups]
-test = [
-  "pytest>=8.3.3",
-  "pytest-cov >=3",
-  "pytest-github-actions-annotate-failures",
-  "sybil>=8.0.0",
-]
+  dev = ["pre-commit>=4.0.1", { include-group = "test" }]
```

I use vscode plugin (https://github.com/ninoseki/vscode-mogami) that shows the latests versions, but that requires the deps to be each on their own line. Something like this would help with that:

```suggestion
  dev = [
    "pre-commit>=4.0.1",
    { include-group = "test" },
  ]
```